reactive/kubernetes_master.py

```diff
@@ -1374,9 +1374,32 @@ def send_data():
                                    key_path=client_key_path)
 
 
+def get_unset_flags(*flags):
```

Comment:
I might actually want to move this into reactive itself, at some point, since it seems generally useful.

Reply:
Maybe I could file a bug for adding this function to `charms.reactive`.